packaging/makerid.py

```diff
@@ -20,10 +21,16 @@
         key = key.replace("tizen.", "")
         key = key.replace("tizen", "")
         if key.strip():
-            rid_list.append(key)
-
-    rid_list = list(set(rid_list))
-    rid_list.sort(reverse=True)
+            if len(key) == 5:
```

Comment:
It doesn't seem like good idea to use the string count of rid(`9.0.0`) to sort `10.0.0`.
How about adding a function to make rid  as integer group(`[10, 0, 0]`) and compare rid as integer.
This will help increase maintainability for future rid, for example 11.0.0, 12.0.0, etc.
```python
def parse_rid_as_int(rid):
    return [int(part) for part in rid.split(".")]
...
rid_list = sorted(rid_list, key=parse_rid_as_int, reverse=True)
```

Reply:
Thank you for your good idea.